pandas/tests/test_optional_dependency.py

```diff
@@ -50,3 +52,11 @@ def test_no_version_raises():
 
     with pytest.raises(ImportError, match="Can't determine .* fakemodule"):
         import_optional_dependency(name)
+
+
+@td.skip_if_installed("tables")
+def test_pytables_raises():
+    df = pd.DataFrame({"A": [1, 2]})
```

Comment:
can you go ahead and create

pandas/tests/io/pytables/test_missing_deps.py and put this there (we need to split test_pytables anyhow)